db/migrate/20171121231237_remove_birthday_from_medicaid_applications.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveBirthdayFromMedicaidApplications < ActiveRecord::Migration[5.1]
```

Comment:
ack! somehow forgot to add this file to my last PR -- sorry! moving too fast over here